NetCoreRestApi/BusinessLayer/Manager/ProductManager.cs

```diff
@@ -0,0 +1,31 @@
+﻿using BusinessLayer.Models;
+using System.Collections.Generic;
+
+namespace BusinessLayer.Manager
+{
+    public class ProductManager : IProductManager
+    {
+        public IEnumerable<ProductModel> GetAll()
+        {
+            List<ProductModel> productModels = new List<ProductModel>();
+            productModels.Add(new StubProductModel("first product"));
+            productModels.Add(new StubProductModel("second product"));
+            productModels.Add(new StubProductModel("third product"));           
+            return productModels;
+        }
+        
+        private class StubProductModel : ProductModel
```

Comment:
One class per one file. Move this class into separated class

Reply:
Done